.github/CODEOWNERS

```diff
@@ -14,67 +14,68 @@
 
-extension/httpforwarder/                   @open-telemetry/collector-contrib-approvers @asuresh4
-extension/observer/                        @open-telemetry/collector-contrib-approvers @asuresh4 @jrcamp
+extension/httpforwarder/                             @open-telemetry/collector-contrib-approvers @asuresh4
+extension/observer/                                  @open-telemetry/collector-contrib-approvers @asuresh4 @jrcamp
 
-internal/awsxray/                          @open-telemetry/collector-contrib-approvers @anuraaga @mxiamxia
-internal/k8sconfig/                        @open-telemetry/collector-contrib-approvers @pmcollins @asuresh4
-internal/splunk/                           @open-telemetry/collector-contrib-approvers @pmcollins @asuresh4
-internal/stanza/                           @open-telemetry/collector-contrib-approvers @djaglowski
+internal/awsxray/                                    @open-telemetry/collector-contrib-approvers @anuraaga @mxiamxia
+internal/k8sconfig/                                  @open-telemetry/collector-contrib-approvers @pmcollins @asuresh4
+internal/splunk/                                     @open-telemetry/collector-contrib-approvers @pmcollins @asuresh4
+internal/stanza/                                     @open-telemetry/collector-contrib-approvers @djaglowski
 
-pkg/batchpertrace/                         @open-telemetry/collector-contrib-approvers @jpkrohling
+pkg/batchpertrace/                                   @open-telemetry/collector-contrib-approvers @jpkrohling
 
-processor/groupbyattrsprocessor/           @open-telemetry/collector-contrib-approvers @pmm-sumo
-processor/groupbytraceprocessor/           @open-telemetry/collector-contrib-approvers @jpkrohling
-processor/k8sprocessor/                    @open-telemetry/collector-contrib-approvers @owais @dmitryax @pmm-sumo
-processor/metricstransformprocessor/       @open-telemetry/collector-contrib-approvers @james-bebbington
-processor/resourcedetectionprocessor/      @open-telemetry/collector-contrib-approvers @jrcamp @pmm-sumo @anuraaga @dashpole
-processor/routingprocessor/                @open-telemetry/collector-contrib-approvers @jpkrohling
-processor/tailsamplingprocessor/           @open-telemetry/collector-contrib-approvers @jpkrohling
+processor/groupbyattrsprocessor/                     @open-telemetry/collector-contrib-approvers @pmm-sumo
+processor/groupbytraceprocessor/                     @open-telemetry/collector-contrib-approvers @jpkrohling
+processor/k8sprocessor/                              @open-telemetry/collector-contrib-approvers @owais @dmitryax @pmm-sumo
+processor/metricstransformprocessor/                 @open-telemetry/collector-contrib-approvers @james-bebbington
+processor/resourcedetectionprocessor/                @open-telemetry/collector-contrib-approvers @jrcamp @pmm-sumo @anuraaga @dashpole
+processor/resourcedetectionprocessor/internal/azure  @open-telemetry/collector-contrib-approvers @mx-psi
```

Comment:
Added myself to CODEOWNERS here (the diff is a bit messy because of spacing changes to keep everything aligned)